src/cogs/forums.py

```diff
@@ -39,7 +39,8 @@ async def done(self, ctx: commands.Context):
 
         if ctx.author == ctx.channel.owner \
                 or permissions.is_staff(ctx.author, ctx.channel) \
-                or permissions.has_role(ctx.author, conf.helpful_role):
+                or permissions.has_role(ctx.author, conf.helpful_role) \
+                or ctx.author in (await ctx.channel.fetch_message(ctx.channel.id)).mentions:
```

Comment:
This also works for normal user created threads actually, meaning that if you ping someone in the opening message of a thread they gain the ability to close the thread. Although this doesn't allow any abuse and is kinda funny since it gives the power of revenge for pinging someone for help, so I'd just keep it this way.

Reply:
I don't like this, it has to look at the message every time.

Instead please check if the author was our bot and look at mentions only then